src/patternfly/components/Compass/compass.scss

```
@@ -92,6 +94,24 @@
   min-height: 0;
 }
 
+.#{$compass}__main-header {
+  & .#{$compass}__panel {
```

The `&` is redundant when using a nested element like this, though you'd want to target the direct descendent in case for some weird reason there were nested panel elements 

```suggestion
  > .#{$compass}__panel {
```